app-catalog/src/components/charts/List.tsx

```diff
@@ -158,25 +159,48 @@ export function ChartsList({ fetchCharts = fetchChartsFromArtifact }) {
         ) : charts.length === 0 ? (
           <Box mt={2} mx={2}>
             <Typography variant="h5" component="h2">
-              {`No charts found for ${
-                search ? `search term: ${search}` : `category: ${chartCategory.title}`
-              }`}
+              {`No charts found for ${search ? `search term: ${search}` : `category: ${chartCategory.title}`}`}
             </Typography>
           </Box>
         ) : (
           charts.map(chart => {
             return (
               <Box maxWidth="30%" width="400px" m={1}>
                 <Card>
-                  <Box height="60px" display="flex" alignItems="center" marginTop="15px">
+                  <Box height="60px" display="flex" alignItems="center" justifyContent="space-between" marginTop="15px">
                     <CardMedia
                       image={`https://artifacthub.io/image/${chart.logo_image_id}`}
                       style={{
                         width: '60px',
                         margin: '1rem',
+                        alignSelf: 'flex-start',
                       }}
                       component="img"
                     />
+                    <Box display="flex" alignItems="center" justifyContent="space-around" marginRight="10px" >
+                      {chart.repository.cncf && (
+                        <Tooltip title="CNCF Project">
+                          <CardMedia
+                            image={`https://github.com/cncf/artwork/blob/1e04eb8355adcf8ffd7bb40ff1da098ea80452e3/other/cncf/icon/color/cncf-icon-color.png?raw=true`}
+                            style={{
+                              width: '30px',
```

Comment:
Please make the other icons 22px and try to make this one match that size.

Reply:
fixed!